src/bin/lpython.cpp

```diff
@@ -241,12 +218,12 @@ int emit_asr(const std::string &infile,
     LCompilers::PassOptions pass_options;
     pass_options.run_fun = "f";
     pass_options.always_run = true;
-    pass_options.verbose = compiler_options.verbose;
-    pass_options.pass_cumulative = compiler_options.pass_cumulative;
-    pass_options.all_symbols_mangling = compiler_options.all_symbols_mangling;
-    pass_options.module_name_mangling = compiler_options.module_name_mangling;
-    pass_options.global_symbols_mangling = compiler_options.global_symbols_mangling;
-    pass_options.intrinsic_symbols_mangling = compiler_options.intrinsic_symbols_mangling;
+    pass_options.verbose = compiler_options.po.verbose;
+    pass_options.pass_cumulative = compiler_options.po.pass_cumulative;
+    pass_options.all_symbols_mangling = compiler_options.po.all_symbols_mangling;
+    pass_options.module_name_mangling = compiler_options.po.module_name_mangling;
+    pass_options.global_symbols_mangling = compiler_options.po.global_symbols_mangling;
+    pass_options.intrinsic_symbols_mangling = compiler_options.po.intrinsic_symbols_mangling;
```

Comment:
I think we need not create `pass_options` here. I think we can reuse the `compiler_options.po` here.